jcl/src/java.base/share/classes/java/lang/invoke/StaticFieldVarHandle.java

```diff
@@ -132,1349 +135,1915 @@
 	 */
 	@SuppressWarnings("unused")
 	static class StaticFieldVarHandleOperations extends VarHandleOperations {
+		private static final JITHelpers jitHelpers = JITHelpers.getHelpers();
+		
+		static void ensureClassIsInitialized(Class<?> clazz) {
```

Comment:
Mark this method as `final` if possible.  It means the JIT won't have to guard the callsites against future class loads

Reply:
Fixed